### DISCUSSION TITLE
plays/shivam-katare/image-gallery

### DISCUSSION THREAD

**Discussion by giscus** [reactplay/react-play#666] [Play Feedback]
# plays/shivam-katare/image-gallery

Its an Image Gallery where users can explore multiple different types of images and on hovering over that they can also, listen about that image.

https://reactplay.io/plays/shivam-katare/image-gallery

**Comment by atapas** [comment]
Great Image Gallery

**Comment by Sachin-chaurasiya** [comment]
Looks great @Shivam-Katare .
